horace_core/sqw/@sqw/smooth.m

```diff
@@ -1,4 +1,4 @@
-function wout = smooth(win, varargin)
+function wout_dnd = smooth(win, varargin)
```

Comment:
not part of your change but next line presuming gataway is gateway, not getaway. Having said which, I still don't know what it means

be good to have explicit description of the input and output arguments, so
input win - sqw without pixels. Although is this still a requirement now the output is a guaranteed dnd separate from any pixel data?
input varargin - parameters for the smooth method for dnds
output wout_dnd - dnd object with image pixels smoothed with respect to the image pixels in input win